model/fieldtypes/DBFloat.php

```diff
@@ -60,3 +60,7 @@ public function prepValueForDB($value) {
 	}
 
 }
+
+if (PHP_MAJOR_VERSION < 7) {
+	include_once 'compat/Float.php';
```

Comment:
What's your solution? Note that putting the check into the file won't block the parse error that PHP7 will throw.